apps/site/pages/docs/headless-cms-integration/2-setting-up-the-vtex-headless-cms.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: '2. Setting up the VTEX Headless CMS'
+sidebar_label: 'Integrating your storefront with VTEX Headless CMS'
+---
+
+import { Callout, Tabs, Tab, Steps } from 'nextra-theme-docs'
+
+
+
+<header>
+# Part 2: Setting up the VTEX Headless CMS in your FastStore project
+</header>
+
+Now that you are familiar with the core concepts of the [VTEX Headless CMS](/docs/headless-cms-overview) and have set it up in your VTEX account, let's get back to our **FastStore** project to define our own **Content Types** and **Sections**.
+
+By the end of this part of this tutorial, you will be able to see your first definition of a Content Type available for use at the VTEX Headless CMS app. Next, we will dive deeper into how to create these definitions in a smoother way.
+
+---
+
+## Before you start
+
+<Steps>
+### Integrate the store with the VTEX Headless CMS
+
+All sections must be available in the Headless CMS so they can be added and managed on your store's pages. To integrate your FastStore project, please refer to the [VTEX Headless CMS integration track](/docs/headless-cms-integration/1-configuring-the-vtex-account).
+
+### Install the FastStore CLI
+
+Make sure to install the FastStore CLI to use its commands locally. Refer to the [FastStore CLI](/docs/getting-started/4-faststore-cli#installing-faststore-cli-globally-using-npm) guide for more information.
+</Steps>
+
+---
+
+## Step by step
+
+### Step 1 - Creating the CMS folder
+
+1. Open the terminal and change to the root directory of your FastStore project.
+2. Create a new folder named `cms` at the root of your FastStore project and a subfolder named `faststore` inside the `cms`. Run the following command to create them:
+
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir -p cms/faststore
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir -p cms\faststore
+  ```
+  </Tab>
+</Tabs>
```

Comment:
@hellofanny and @caroolcanelas, I believe we can keep the code snippets for folders and files if we follow a similar approach similar to this example. Before, we asked the person to create the `cms` folder, then we gave the code to run and create the `faststore` subfolder. Now, with the `mkdir -p cms/faststore`, the person creates both simultaneously.

Reply:
sounds good.